docs/docs/08-guides/01-terraform-backend.md

```diff
@@ -0,0 +1,114 @@
+
+The local state works perfectly for small scale, individual, or testing projects where there's no need for collaboration or sharing of state data. It is simple and doesn't require any additional setup or configuration to use. However, it lacks many features required for team-based work or production-level infrastructure.
+
+The local state is stored unencrypted on your local filesystem and is not versioned, meaning you could lose or overwrite your state file without any way to recover it. Also, it doesn't support state locking, so concurrent runs could potentially corrupt your state file. These shortcomings could lead to inconsistencies between your real infrastructure and your Terraform configuration, leading to potential infrastructure drift.
+
+The local state backend is a great way to get started with Terraform, but for production workloads, especially where team collaboration is required, a remote backend like S3 is generally a better option. This is why we will be focusing on setting up a Terraform [S3 backend](https://developer.hashicorp.com/terraform/language/settings/backends/s3) in the next section of this document.
+
+# Transitioning to an S3 Backend
+
+The default local state storage is useful for getting started with Terraform, but as your infrastructure grows, or if you need to collaborate with a team, the local backend becomes less suitable. To handle such situations, Terraform supports [multiple backend types](https://developer.hashicorp.com/terraform/language/settings/backends/configuration), one of which is Amazon S3.
+
+Storing your Terraform state file on an S3 backend allows your team to share access to the state and collaborate efficiently, maintaining a consistent view of the infrastructure. It also enables state versioning, preserving, retrieving, and restoring every version of your state file. This versioning feature of S3 provides a means to roll back to a previous state if necessary.
+
+In addition, an S3 backend offers state locking via a DynamoDB table, preventing concurrent modifications of the state that could lead to conflicts and potential data loss. S3's high durability and availability ensure that your state data is resilient, safe, and accessible whenever you need it. Furthermore, by using Amazon S3, you can leverage server-side encryption to secure your state data, providing an additional layer of security.
+
+In the following sections, we will guide you through the process of setting up an S3 backend for your Terraform configurations, empowering you to handle larger, more complex, and collaborative infrastructure projects.
+
+# Setting up an S3 Terraform Backend
```

Comment:
Can we turn this into 

```sh
wing bootstrap -t tf-aws aws://account/region
```

Would have been sooooo much nicer!

Reply:
Good point. I'll create a follow up issue for a bootstrap command